ktypes/preempt-rt/preempt-rt.cfg

```diff
@@ -1,2 +1,23 @@
 # Processor type and features
 CONFIG_PREEMPT_RT=y
+
+# Use performance CPU frequency governor to decrease latency when
+# switching from idle to active task
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+# CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND is not set
+CONFIG_CPU_FREQ_STAT_DETAILS=y
+
+# Make network subsystem ready to realtime working on boot
+CONFIG_NET_ACT_SKBEDIT=y
+CONFIG_NET_PKTGEN=y
+CONFIG_NET_SCH_MULTIQ=y
+CONFIG_NET_SCH_CBS=y
+CONFIG_NET_SCH_ETF=y
+CONFIG_NET_SCH_TAPRIO=y
+CONFIG_NET_SCH_MQPRIO=y
+CONFIG_NET_SCH_INGRESS=y
+
+# Enable the generic software mailbox
+CONFIG_GENERIC_SOFTWARE_MAILBOX=y
+# Enable virtio performance driver
+CONFIG_VIRTIO_TRANS=y
```

Comment:
These options will also be enabled on x86, but I guess they should all be fine, right?

Reply:
@ricardosalveti thanks for noticing it, I'm making these options enabled only for imx machines. We can't rely on these features to correct work in rt for all other machines even if these machines support these features.

Reply:
@ricardosalveti Actually, CONFIG_VIRTIO_TRANS is just for use in test tools, not a real feature :) So I removed it at all.
An imx-specific feature is separated.